python-pyproject-toml/snakesay-project/snakesay/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""Snakesay - cowsay, but with a snake."""
+
+__version__ = "0.1.0"
```

Comment:
We're using `"1.0.0"` in the tutorial. Should we bump this one as well?

Reply:
Why not!